politeiad/backend/gitbe/gitbe.go

```diff
@@ -2616,7 +2623,7 @@ func (g *gitBackEnd) rebasePR(id string) error {
 }
 
 // New returns a gitBackEnd context.  It verifies that git is installed.
-func New(anp *chaincfg.Params, root string, dcrtimeHost string, gitPath string, id *identity.FullIdentity, gitTrace bool) (*gitBackEnd, error) {
+func New(anp *chaincfg.Params, root string, dcrtimeHost string, gitPath string, id *identity.FullIdentity, gitTrace bool, cmsMode bool) (*gitBackEnd, error) {
```

Comment:
Instead of using a `cmsMode bool` argument, this should be a generic `mode string` argument so that additional modes can be added in the future.  A switch statement can be used to handle the mode specific setup.